### PR TITLE
CON-2050: added userSignInActivity Information

### PR DESCRIPTION
**Pull request by hitarth-sumo**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 1]
```diff
@@ -135,7 +141,12 @@ To configure a Microsoft Azure AD Inventory Source:
 
 10. **Supported APIs to collect**. Select one or more of the available APIs: **Devices** and **Users**.
 
-11. When you are finished configuring the Source, click **Submit**.
+11. **Collect Users SignInActivity Data**. By enabling the checkbox, signInActivity information will also be included in the user response. [Reference](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time)
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:144]:
```suggestion
11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 2]
```diff
@@ -174,7 +185,7 @@ Azure AD Inventory Source.
 | `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable
 | `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
 | `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: Devices, and Users. For example, for both you'd use: ["Devices","Users"] | modifiable |
-
+| `userSignInActivity` | Boolean | No | False | Select the checkbox to include signInActivity data for user | modifiable |
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:190]:
```suggestion
| `userSignInActivity` | Boolean | No | False | Select the checkbox to include the sign in activity information in your response. | modifiable |
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 3]
```diff
@@ -95,6 +95,12 @@ From there, select (or search for) the following permissions under type **Applic
 | User    | Application (work or school account) | User.Read.All, Directory.ReadAll   |
 | Devices | Application (work or school account) | Device.Read.All, Directory.ReadAll |
 
+To collect signInActivityData for User additional permission will be required.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:98]:
```suggestion
You require additional permission to collect `signInActivityData`.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 4]
```diff
@@ -135,7 +141,12 @@ To configure a Microsoft Azure AD Inventory Source:
 
 10. **Supported APIs to collect**. Select one or more of the available APIs: **Devices** and **Users**.
 
-11. When you are finished configuring the Source, click **Submit**.
+11. **Collect Users SignInActivity Data**. By enabling the checkbox, signInActivity information will also be included in the user response. [Reference](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time)
+- There are two prerequisites required to collect the signInActivity information.
+    - An Azure AD Premium P1/P2 license.
+    - Read all audit log data[AuditLog.Read.All] permission.
```
[the remark this answers is not in the document]
Comment (JV0812) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:147]: @hitarth-sumo information about "Read all audit log data[AuditLog.Read.All] permission" is already mentioned in Set up a Service Application.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 5]
```diff
@@ -174,7 +187,7 @@ Azure AD Inventory Source.
 | `secret_key` | String | Yes |  | Provide the Application Client Secret Value you created in Azure. | modifiable
 | `application_id` | String | Yes |  | Provide the Application (client) ID you got after you registered (created) the Azure Application. | modifiable |
 | `supported_apis` | Array of strings | Yes |  | Define one or more of the available APIs to collect: Devices, and Users. For example, for both you'd use: ["Devices","Users"] | modifiable |
-
+| `userSignInActivity` | Boolean | No | False | Select the checkbox to include signInActivity data for user | modifiable |
 Microsoft Azure AD Inventory Source JSON example:
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:191]: Add a space between line 190 and 191. Add a H3 title before line 191 as JSON example.

**Review by JV0812 — CHANGES_REQUESTED** [review]
Minor fixes.

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md (hunk 1 above)
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md]: Done

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md]: Done

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md (hunk 2 above)
Comment (hitarth-sumo) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md]: Done

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 6]
```diff
@@ -135,7 +141,15 @@ To configure a Microsoft Azure AD Inventory Source:
 
 10. **Supported APIs to collect**. Select one or more of the available APIs: **Devices** and **Users**.
 
-11. When you are finished configuring the Source, click **Submit**.
+11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
+
+:::note
+  To collect the `signInActivity` information you require `Azure AD Premium P1/P2 license` and `Read all audit log data[AuditLog.Read.All] permission`.
```
Comment (JV0812) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:147]:
```suggestion
  To collect the `signInActivity` information you should have `Azure AD Premium P1/P2 license`.
```

**Review by JV0812 — APPROVED** [review]
LGTM

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md [hunk 7]
```diff
@@ -135,7 +141,15 @@ To configure a Microsoft Azure AD Inventory Source:
...
 
-11. When you are finished configuring the Source, click **Submit**.
+11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
+
+:::note
+  To collect the `signInActivity` information you should have `Azure AD Premium P1/P2` license.
+:::
+
+12. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
+
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/microsoft-azure-ad-inventory-source.md:151]:
```suggestion
11. **Collect Users SignInActivity Data**. By enabling the checkbox, you can also include the sign in activity information in your user response. [Learn more](https://learn.microsoft.com/en-us/graph/api/user-list?view=graph-rest-1.0&tabs=http#example-10-get-users-including-their-last-sign-in-time).
   :::note
   To collect the `signInActivity` information you should have `Azure AD Premium P1/P2` license.
   :::
12. **Processing Rules for Logs**. Configure any desired filters, such as allowlist, denylist, hash, or mask, as described in [Create a Processing Rule](/docs/send-data/collection/processing-rules/create-processing-rule).
```

**Review by kimsauce — APPROVED** [review]

**Pull request merged**